src/pages/api/web-scraping/flights.ts

```diff
@@ -36,6 +36,7 @@ export default async function getFlights(req: NextApiRequest, res: NextApiRespon
   }
 
   // Retrieve analysis event from the Server API using the request ID
+  // @ts-ignore TODO: fix types
```

Comment:
Don't forget

Reply:
Thank you, that's related to my question here actually: https://fingerprintjs.slack.com/archives/C050TKU9X5L/p1700666530484769

Reply:
@ilfa fixed thank you!